src/EnergyPlus/ManageElectricPower.cc

```diff
@@ -2457,7 +2463,7 @@ namespace ManageElectricPower {
 		// Flow
 
 		ElecProducedCoGen = GetInstantMeterValue( ElecProducedCoGenIndex, 2 ); //whole building
-		ElecProducedFacility = ElecProducedCoGen + WholeBldgElectSummary.ElecProducedPVRate * TimeStepSys * SecInHour + WholeBldgElectSummary.ElecProducedWTRate * TimeStepSys * SecInHour; //whole building
+		ElecProducedFacility = ElecProducedCoGen + ( WholeBldgElectSummary.ElecProducedPVRate + WholeBldgElectSummary.ElecProducedWTRate + WholeBldgElectSummary.ElecProducedStorageRate ) * TimeStepSys * SecInHour; //whole building
```

Comment:
I am a little uncomfortable with adding in the storage term here, can you explain the thinking here?